modules/apps.nix

```diff
@@ -34,6 +34,17 @@ let
           A path to an executable or a derivation with `meta.mainProgram`.
         '';
       };
+      meta = mkOption {
+        type = types.lazyAttrsOf lib.types.raw;
+        default = { };
```

Comment:
In hindsight, should we add a default here? Taking from `program`, that is, if it is a derivation. But `coercedTo` seems lossy, so this probably is not possible.

Until then, users basically will have to explicit set it like follows:

<img width="440" alt="image" src="https://github.com/user-attachments/assets/a3dc54f8-f8b5-45f1-be81-460956c4530c">